packages/deployment/README.md

```diff
@@ -31,7 +31,7 @@ For example:
 | field | default     | description
 | ----- | ----------- | -----------
 | stackName | (required) | the name used as a prefix in all aws resources
```

Comment:
`stackName` is no longer required, is it? But `prefix` would be?

Reply:
Good catch 👍